interfaces/builtin/screen_inhibit_control.go

```diff
@@ -58,16 +58,16 @@ dbus (send)
 # freedesktop.org ScreenSaver
 dbus (send)
     bus=session
-    path=/{,org/freedesktop/,org.gnome/}Screensaver
-    interface=org.freedesktop.ScreenSaver
-    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit,SimulateUserActivity}
+    path=/{,org/freedesktop/,org.gnome/}Screen{s,S}aver
+    interface=org.freedesktop.Screen{s,S}aver
+    member=org.freedesktop.Screen{s,S}aver.{Inhibit,UnInhibit,SimulateUserActivity}
     peer=(label=unconfined),
 
 # gnome, kde and cinnamon screensaver
 dbus (send)
     bus=session
-    path=/{,ScreenSaver}
-    interface=org.{gnome.ScreenSaver,kde.screensaver,cinnamon.ScreenSaver}
+    path=/{,Screensaver,ScreenSaver}
+    interface=org.{gnome.Screensaver,gnome.ScreenSaver,kde.screensaver,cinnamon.ScreenSaver}
```

Comment:
I don't recall seeing `org.gnome.Screensaver`. I also tried searching the history of gnome-shell an g-s-d- repos for commits that fix this typo bu found nothing in particular. Probably it's worth double checking with someone from GNOME, but my guess is you can drop `org.gnome.Sceensaver` variant.

Reply:
It may not be libgtk; it could simply be a client that used it. The policy was written by testing clients that use the API. Unfortunately, I don't recall which client used Screensaver. Even if it was just a passing version of a library that is since fixed, we don't know what snaps are embedding and since we released with Screensaver, we cannot revert it in series 16 without risking breaking snaps in the wild, which is why I used the alternation instead of a correction.

Reply:
I looked into this more. It was fixed in https://github.com/snapcore/snapd/pull/3531 to address https://forum.snapcraft.io/t/possible-screen-inhibit-control-bug/876. The change was:

https://github.com/snapcore/snapd/pull/3531/commits/d3af006501948662fc6dda9b0149e0695966740d

```
 # freedesktop.org ScreenSaver
 dbus (send)
     bus=session
-    path=/Screensaver
+    path=/{,org/freedesktop/,org.gnome/}Screensaver
     interface=org.freedesktop.ScreenSaver
     member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit,SimulateUserActivity}
     peer=(label=unconfined),
```

In other words, what was tested in the past was /Screensaver and the above tried to address the Mr Rescue issue, but actually did not. In this light, I will update this PR. Thanks for checking on this!